src/ast/mod.rs

```diff
@@ -430,7 +432,13 @@ pub enum GenericCommand<Head, Leaf> {
     ///          :when ((= a (Num 0)))
     /// ```
     ///
-    Rewrite(Symbol, GenericRewrite<Head, Leaf, ()>),
+    /// Add the`:subsume` flags to apply those to the left hand side on a rewrite when it's mtched.
+    ///
+    /// ```text
+    /// (rewrite (Mul a 2) (bitshift-left a 1) :subsume)
```

Comment:
Could you show the desugared version here, and link to the documentation for subsume? (see prev comment)

Reply:
Thanks for catching this! Just added a desugared version and linked to the `Change`/